lib/coursemology/evaluator/models/programming_evaluation/package.rb

```diff
@@ -0,0 +1,7 @@
+class Coursemology::Evaluator::Models::ProgrammingEvaluation::Package
+  # Constructs a new Package.
+  #
+  # @param [IO] io The IO object containing the package data.
+  def initialize(io)
```

Comment:
@allenwq I'll ignore this first, I want to implement the logic in a separate PR.